packages/sdk/src/client/client.ts

```diff
@@ -871,6 +911,46 @@ export class Client {
         logger.info(`[WD] c:"${this.getKey()}" watches d:"${docKey}"`);
 
         return new Promise((resolve, reject) => {
+          let idleTimer: any;
```

Comment:
I used the type any because `NodeJS.Timeout` caused errors in other packages, such as `TS2503: Cannot find namespace 'NodeJS'`. Any ideas on how to fix this?

Reply:
We can use the utility type `ReturnType` rather than using an environment-dependent type like `NodeJS.Timeout`.

```suggestion
          let idleTimer: ReturnType<typeof setTimeout>;
```

Reply:
We can use the utility type `ReturnType` rather than using an environment-dependent type like `NodeJS.Timeout`.

```suggestion
          let idleTimer: ReturnType<typeof setTimeout>;
```

Reply:
Thanks for your suggestion! I have committed your suggestion.